### PR TITLE
Add PyCharm banner to homepage to boost fundraising

### PR DESCRIPTION
**Pull request by frankwiles**
Originally I added the `header_content` block and noticed the project uses hyphens so fixed that. 

Added homepage banner which needed a separate image for visual reasons.

### CODE REVIEW

File: djangoproject/templates/base.html [hunk 1]
```diff
@@ -49,13 +49,13 @@
 
     {% include "includes/header.html" %}
 
-    {% block header_container %}
```
Comment (MarkusH) [inline djangoproject/templates/base.html]: Do you mind leaving that as-is, please :)

**Comment by timgraham** [comment]
It doesn't look so good at a reduced window width.

reduced-width

File: djangoproject/templates/base.html (hunk 1 above)
Comment (timgraham) [inline djangoproject/templates/base.html]: I see both ways, "layout_class",  "content-related" ... don't care much myself. Django seems to use both too.

**Comment by timgraham** [comment]
I think it would be more effective to put it in the light green "Documentation" header. Probably more people visit that than the home page.

File: djangoproject/templates/base.html (hunk 1 above)
Comment (frankwiles) [inline djangoproject/templates/base.html]: Ah ok I saw a ton of hyphens so I thought I was breaking the convention.  Happy to revert that!

**Comment by frankwiles** [comment]
Good idea, I'll do that instead!

**Comment by frankwiles** [comment]
Ok all fixed. I think this looks much better anyway! 
screen shot 2016-07-13 at 4 27 46 pm

**Comment by timgraham** [comment]
You prefer it there rather than in the documentation header? I don't think many developers visit the home page compared to the documentation. Anyway, I can merge this for now.

**Comment by timgraham** [comment]
merged in 7a3f7daf5b626636a8926b61c3d24f48eee44f9a.

**Comment by olasitarska** [comment]
I think adding this info on documentation pages as @timgraham suggested is a great idea!

**Comment by frankwiles** [comment]
Ah that's what you meant by "documentation header" :) I think that's a great idea.  I'll do that now.

**Comment by frankwiles** [comment]
Hmmm I keep getting errors while trying to build the documentation. 

```
Exception occurred:
  File "/Users/frank/.virtualenvs/djangoproject.com/lib/python3.5/site-packages/docutils/nodes.py", line 1367, in set_duplicate_name_id
    old_node = self.ids[old_id]
KeyError: 'philosophy'
The full traceback has been saved in /var/folders/n1/1_ncw7sx4vv_ts464gmyrbbm0000gn/T/sphinx-err-u1xjye62.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Traceback (most recent call last):
  File "./manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/frank/.virtualenvs/djangoproject.com/lib/python3.5/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "/Users/frank/.virtualenvs/djangoproject.com/lib/python3.5/site-packages/django/core/management/__init__.py", line 345, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/frank/.virtualenvs/djangoproject.com/lib/python3.5/site-packages/django/core/management/base.py", line 348, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/frank/.virtualenvs/djangoproject.com/lib/python3.5/site-packages/django/core/management/base.py", line 399, in execute
    output = self.handle(*args, **options)
  File "/Users/frank/work/src/djangoproject.com/docs/management/commands/update_docs.py", line 101, in handle
    str(build_dir),         # Destination directory
  File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/subprocess.py", line 584, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['sphinx-build', '-b', 'json', '-D', 'language=fr', '-q', '/Users/frank/work/src/djangoproject.com/data/djangodocs/sources/1.6/docs', '/Users/frank/work/src/djangoproject.com/data/djangodocs/fr/1.6/_build/json']' returned non-zero exit status 1
```

Any ideas on the fix? Updated gettext and using current master :(

**Comment by timgraham** [comment]
It's a known issue (#627). For testing, you don't need to build all versions anyway.

**Pull request closed without merging**